src/components/Select.vue

```diff
@@ -571,7 +571,12 @@ export default {
      */
     selectOnKeyCodes: {
       type: Array,
-      default: () => [13],
+      default: () => [
+        // enter
+        13,
+        // space
+        32,
```

Comment:
I think space must not default. Consider if we want to search an option that contain space.